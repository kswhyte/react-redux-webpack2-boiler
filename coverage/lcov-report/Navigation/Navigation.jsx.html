<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for Navigation/Navigation.jsx</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">Navigation</a> Navigation.jsx
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>3/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>3/3</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">/*eslint no-unused-vars: */
&nbsp;
import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';
&nbsp;
const Navigation = props =&gt; {
  Navigation.propTypes = {
    title: PropTypes.string,
    children: PropTypes.object
  };
&nbsp;
  return (
    &lt;nav className="navigation-wrapper"&gt;
      &lt;h1&gt;Navigation&lt;/h1&gt;
      &lt;Link to="/login"&gt;Login&lt;/Link&gt;
      &lt;Link to="/patientsearch"&gt;Search for Patients&lt;/Link&gt;
      &lt;Link to="/users"&gt;User Management Page&lt;/Link&gt;
      &lt;Link to="/patientinfo"&gt;Confirm Patient Info&lt;/Link&gt;
      &lt;Link to="/searchandcalendar"&gt;Search and Calendar&lt;/Link&gt;
      &lt;Link to="/confirm"&gt;Confirm&lt;/Link&gt;
    &lt;/nav&gt;
  );
};
&nbsp;
export default Navigation;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Wed Aug 09 2017 12:07:41 GMT-0600 (MDT)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
</body>
</html>
