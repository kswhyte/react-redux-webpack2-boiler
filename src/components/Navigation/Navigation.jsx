/*eslint no-unused-vars: */

import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

const Navigation = props => {
  Navigation.propTypes = {
    title: PropTypes.string,
    children: PropTypes.object
  };

  return (
    <nav className="navigation-wrapper">
      <h1>Navigation</h1>
      <Link to="/login">SignOn</Link>
      <Link to="/patientsearch">Search for Patients</Link>
      <Link to="/users">User Management Page</Link>
      <Link to="/patientinfo">Confirm Patient Info</Link>
      <Link to="/searchandcalendar">Search and Calendar</Link>
      <Link to="/confirm">Confirm</Link>
    </nav>
  );
};

export default Navigation;
