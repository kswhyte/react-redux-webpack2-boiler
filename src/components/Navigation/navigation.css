@import "imports/_helpers.css";
@import "imports/_layout.css";
@import "imports/_form-elements.css";
@import "imports/_typography.css";

.navigation-wrapper {
}
