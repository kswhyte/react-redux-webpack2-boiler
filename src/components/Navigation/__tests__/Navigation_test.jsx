import React from 'react';
// import { storiesOf, action, linkTo } from '@kadira/storybook';
import Navigation from '../Navigation';

// import { specs, describe, it } from 'storybook-addon-specifications';

import {storiesOf, describe, it, specs
} from "../../../../.storybook/facade";

import {shallow} from 'enzyme';
import expect from 'expect';


const stories = storiesOf('Navigation', module);

stories.add('Story', () => {
    const navigationStory = (
      <Navigation/>);

    specs(() => describe('Show a successful alert', () => {
        it('Should render the Navigation component without crashing', () => {
            let output = shallow(navigationStory);
            expect(output.find('.navigation-wrapper').length).toEqual(1);
        });
    }));
    return navigationStory;
});
