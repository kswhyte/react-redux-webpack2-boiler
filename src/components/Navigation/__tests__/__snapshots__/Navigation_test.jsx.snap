// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Story 1`] = `
<nav
  className="navigation-wrapper"
>
  <h1>
    Navigation
  </h1>
  <a
    href="/login"
    onClick={[Function]}
  >
    SignOn
  </a>
  <a
    href="/patientsearch"
    onClick={[Function]}
  >
    Search for Patients
  </a>
  <a
    href="/users"
    onClick={[Function]}
  >
    User Management Page
  </a>
  <a
    href="/patientinfo"
    onClick={[Function]}
  >
    Confirm Patient Info
  </a>
  <a
    href="/searchandcalendar"
    onClick={[Function]}
  >
    Search and Calendar
  </a>
  <a
    href="/confirm"
    onClick={[Function]}
  >
    Confirm
  </a>
</nav>
`;
